benches/generators.rs

```diff
@@ -66,15 +66,22 @@ gen_uint!(gen_u64_chacha, u64, ChaChaRng);
 gen_uint!(gen_u64_std, u64, StdRng);
 gen_uint!(gen_u64_os, u64, OsRng);
 
+#[bench]
+fn gen_u64_jitter(b: &mut Bencher) {
+    let mut rng = JitterRng::new().unwrap();
+    b.iter(|| {
+        black_box(rng.gen::<u64>());
+    });
+    b.bytes = size_of::<u64>() as u64 * RAND_BENCH_N;
+}
+
 macro_rules! init_gen {
     ($fnn:ident, $gen:ident) => {
         #[bench]
         fn $fnn(b: &mut Bencher) {
-            let mut rng = XorShiftRng::new().unwrap();
+            let mut rng = OsRng::new().unwrap();
```

Comment:
I think this uses `XorShiftRng` because we want to benchmark `from_rng`, not the `OsRng` itself

Reply:
That was my first intention with this benchmark. Somewhere along the road it seemed to make sense to do the complete measurement. Ah, yes, to compare JitterRng against. I don't think it matters all that much though.

Reply:
That makes sense specifically for `JitterRng` but not for the others.

Reply:
I can reverse it, together with fixes for the CI failures